spec/controllers/spree/api/social_auths_controller_spec.rb

```diff
@@ -1,7 +1,23 @@
 require 'spec_helper'
 
 describe Spree::Api::SocialAuthsController do
-  it 'holi' do
-    expect(post: '/api/facebook/signin').to route_to(controller: 'socials', action: 'signin')
+  it 'offers endpoint for amazon' do
+    expect(post: '/api/amazon/signin').to route_to(format: 'json', controller: 'spree/api/social_auths', action: 'amazon_signin')
```

Comment:
can this be done using the helpers? like

`post: amazon_signin_path` or something? if not, it's ok